src/configure/test/runTest.ts

```diff
@@ -0,0 +1,23 @@
+import * as path from 'path';
+
+import { runTests } from 'vscode-test';
+
+async function main() {
+	try {
+		// The folder containing the Extension Manifest package.json
+		// Passed to `--extensionDevelopmentPath`
+		const extensionDevelopmentPath = path.resolve(__dirname, '../../');
+
+		// The path to test runner
+		// Passed to --extensionTestsPath
+		const extensionTestsPath = path.resolve(__dirname, './suite/index');
```

Comment:
can we pass the test file or the test name which we want to run? This can be useful when we are trying to fix one specific test and don't want to run all tests all the time

Reply:
For running a particular test, we can use the command `mocha -g 'test_name'`.